src/main/scala/mesosphere/marathon/api/RootApplication.scala

```diff
@@ -0,0 +1,18 @@
+package mesosphere.marathon
+package api
+
+import javax.ws.rs.core.Application
+import javax.ws.rs.ext.ExceptionMapper
+import scala.collection.JavaConverters._
+
+/**
+  * Really simple JAX Application which encompasses a list of singletons
```

Comment:
Singletons of what? Could you add `@param` descriptions?

Reply:
Will do

Reply:
✅